gradle.properties

```diff
@@ -0,0 +1 @@
+org.gradle.jvmargs=-Xmx2g -XX:MaxPermSize=512m
```

Comment:
GradleCeption is green (see the build linked below). We fixed the daemon killing a while ago.